src/Plugin/monitoring/SensorPlugin/TidetimesSensorPlugin.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\tide_core\Plugin\monitoring\SensorPlugin;
+
+use Drupal\monitoring\Result\SensorResultInterface;
+use Drupal\monitoring\SensorPlugin\ExtendedInfoSensorPluginInterface;
+use Drupal\monitoring\SensorPlugin\SensorPluginBase;
+
+/**
+ * Monitors the Section API connection.
+ *
+ * @SensorPlugin(
+ *   id = "tide_times_sensor",
+ *   label = @Translation("Tide Times Sensor"),
+ *   description = @Translation("Monitors tide modules."),
+ * )
+ */
+class TidetimesSensorPlugin extends SensorPluginBase implements ExtendedInfoSensorPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function resultVerbose(SensorResultInterface $result) {
+    return [
+      '#type' => 'inline_template',
+      '#template' => '
+      <div class="system-info-collapsible">
+        {% for section, data in sections %}
+          <div class="collapsible-section">
+            <h3">{{ section|replace({"_": " "})|title }}</h3>
+            <div style="display: block;">
+                <ul>
+                  {% for key, value in data %}
+                    <li><strong>{{ key }}:</strong> {{ value }}</li>
+                  {% endfor %}
+                </ul>
+            </div>
+          </div>
+        {% endfor %}
+      </div>
+      
+      <style>
+        .system-info-collapsible h3 {
+          cursor: pointer;
+          background-color: #f0f0f0;
+          padding: 10px;
+          margin: 5px 0;
+        }
+        .system-info-collapsible h3:hover {
+          background-color: #e0e0e0;
+        }
+        .system-info-collapsible ul {
+          margin-left: 20px;
+        }
+      </style>
```

Comment:
Not necessary to do it now, but in future if this grows may be we can separate it in twig template and also separate the style and load it as library for better readability.